osquery/core/init.cpp

```diff
@@ -603,7 +603,7 @@ void Initializer::start() const {
     if (tool_ == ToolType::DAEMON) {
       LOG(WARNING) << message;
     } else {
-      LOG(INFO) << message;
```

Comment:
`INFO` is silenced on the shell by default, what is the thought here, why the change?